### DISCUSSION TITLE
follow relationship, multiple fields point to the same model

### DISCUSSION THREAD

**Discussion by ghost** [fastapi/sqlmodel#657] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Optional
from datetime import datetime

from sqlalchemy import UniqueConstraint
from sqlmodel import Field, SQLModel


class FollowersBase(SQLModel):
    id: Optional[int] = Field(default=None, primary_key=True)
    following: Optional[int] = Field(nullable=False, foreign_key="user.id")
    followers: Optional[int] = Field(nullable=False, foreign_key="user.id", )


class Followers(FollowersBase, table=True):
    ...


class FollowersRead(FollowersBase):
    ...
```

### Description

user: Optional[User] = Relationship(back_populates="")

How do I fill this in? I have two fields pointing to user.
Thank you for your help

### Operating System

macOS

### SQLModel Version

0.0.8

### Python Version

Python 3.10.8

**Comment by enchance** [comment]
Did you ever find the answer? Have the same problem.
